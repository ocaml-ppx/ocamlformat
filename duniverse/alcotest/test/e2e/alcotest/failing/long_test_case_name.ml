let () =
  let open Alcotest in
  run __FILE__
    [
      ( "test-a",
        [
          test_case "Passing" `Quick (fun () -> ());
          test_case
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit. \
             Suspendisse mollis, orci sed venenatis efficitur, eros est \
             imperdiet purus, sit amet tincidunt massa diam ut elit."
            `Quick (fun () -> Alcotest.fail "Failed");
        ] );
    ]
