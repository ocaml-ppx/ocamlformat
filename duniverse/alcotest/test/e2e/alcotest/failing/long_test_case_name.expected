Testing `test/e2e/alcotest/failing/long_test_case_name.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Passing.
> [FAIL]        test-a          1   Lorem ipsum dolor sit amet, consectetur a...

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]        test-a          1   Lorem ipsum dolor sit amet, consectetu...  │
└──────────────────────────────────────────────────────────────────────────────┘
ASSERT Failed
FAIL Failed

Logs saved to `<build-context>/_build/_tests/<test-dir>/test-a.001.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 2 tests run.
